middleman_protocol/middleman_protocol/constants.py

```diff
@@ -66,3 +67,6 @@ class ErrorCode(IntEnum):
 
 # When deserializing message is not possible, response should use following request_id
 REQUEST_ID_FOR_RESPONSE_FOR_INVALID_FRAME = 0
+
+# Interval in seconds, after which MiddleMan should send HeartbeatFrame
+HEARTBEAT_INTERVAL = 15
```

Comment:
Shouldn't be in `middleman.constants`?

Reply:
Good question. I would agree, but this interval is (in a way) part of `HeartbeatFrame`... Ok, I will move it.